forch/forchestrator.py

```diff
@@ -816,7 +817,7 @@ def _validate_config(self, config):
                 is_stack = 1 if 'stack' in if_obj else 0
                 is_access = 1 if 'native_vlan' in if_obj else 0
                 is_tap = 1 if if_obj['description'] == 'tap' else 0
-                is_mirror = 1 if if_obj['description'] == 'mirror' else 0
+                is_mirror = 1 if if_obj['description'].lower() == 'mirror' else 0
```

Comment:
is this strictly necessary?  Generally it's better to be more exact in matching, not permissive. Why is there a need for this to handle both MIRROR, mirror, MiRRor, etc... ?

Reply:
Updated to "MIRROR". That's what are using in productin...

Reply:
Btw, should we change "tap" to "TAP' to match the style in production? There we always uppercase prefixes for all the port descriptions.